src/plugins/vis_type_timeseries/public/application/components/panel_config/table.js

```diff
@@ -115,13 +116,19 @@ export class TablePanelConfig extends Component {
                       />
                     }
                   >
-                    <FieldSelect
-                      fields={this.props.fields}
-                      value={model.pivot_id}
-                      indexPattern={model.index_pattern}
-                      onChange={this.handlePivotChange}
-                      fullWidth
-                    />
+                    <VisDataContext.Consumer>
```

Comment:
just interesting, maybe we should pass consumer into FieldSelect component and remove uiRestrictions from props. It guarantee that we apply it for all needed places.

@sulemanof  what do you think?

Reply:
We checked it offline with @VladLasitsa. Looks like we have at least two places where fields should not be filtered by `uiRestrictions`
- src/plugins/vis_type_timeseries/public/application/components/annotations_editor.js
- src/plugins/vis_type_timeseries/public/application/components/vis_types/table/config.js